src/main/java/com/redhat/insights/agent/AgentSubreportSerializer.java

```diff
@@ -18,6 +18,9 @@ public void serialize(
     generator.writeStartObject();
     generator.writeStringField("version", subreport.getVersion());
     generator.writeStringField("workloadType", subreport.getGuessedWorkload());
+    generator.writeStringField("is_ocp", subreport.isOCP());
+    generator.writeStringField("pod_name", subreport.getPodName());
+    generator.writeStringField("pod_namespace", subreport.getPodNamespace());
```

Comment:
Would it make sense to include these fields only if `is_ocp` is true?

Reply:
Well, normally I'd say to always submit them so that the other side won't have to worry about them not being there. On the other side though, since it might have to deal with multiple versions of the reports, it has to check for them anyway. I'm not sure it makes a difference one way or the other.